iidm/iidm-impl/src/test/java/com/powsybl/iidm/network/impl/extensions/LoadDetailTest.java

```diff
@@ -24,7 +26,7 @@
  */
 public class LoadDetailTest {
```

Comment:
Maybe this test could be part of the TCK and have a subclass in each implementation?